### ISSUE TITLE
Add additional arrowheads to the interface

### ISSUE THREAD

**Issue by mtolmacs** [excalidraw/excalidraw#8611]
There are more types of arrowheads implemented than what's available on the interface. In order to add them to the interface there should be a `more...`/`less...` link should allow showing/hiding additional arrowheads in the arrowhead popout.
![Image](https://github.com/user-attachments/assets/d7f3eed6-7508-468f-95ff-dd1761cb7c7b)

* Add the show/hide logic to the arrowhead popout
* Add the additionally implemented arrowheads to the conditionally hidden section
* Move the T arrow end to the conditionally hidden section
* Add "crow's feet" arrowhead and put it in the hidden section

**Comment by mtolmacs** [comment]
`packages/excalidraw/element/types.ts` contains the `type Arrowhead` which has the names for the currently hidden arrowheads.

**Comment by j-krolak** [comment]
I would like to work on it. To make sure I understand you correctly, by "crow's feet" arrowhead you mean something like this 
![Image](https://github.com/user-attachments/assets/b8d0f6e2-e230-4ac5-ab7b-85a1663389d8)

**Comment by mtolmacs** [comment]
@j-krolak yes, exactly on the "crow's feet". Please link a PR (even if it's draft) so everybody can follow your work.

**Comment by j-krolak** [comment]
Hey @mtolmacs , I've made a PR https://github.com/excalidraw/excalidraw/pull/8616  which resolves this issue.
